Scripts/seed.py

```
@@ -0,0 +1,448 @@
+import requests
+import sys
+import zipfile
+
+
+def tenant_exists(admin_bff_url, tenant_name):
+    # 1. GET {admin-bff}/api/Facility/{tenant-name}
+    check_url = f"{admin_bff_url}/api/Facility/{tenant_name}"
+    print(f"Checking if tenant '{tenant_name}' exists at {check_url}...")
+
+    try:
+        response = requests.get(check_url)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)